### ISSUE TITLE
Scrollable Footer

### ISSUE THREAD

**Issue by liKe2k1** [wenzhixin/bootstrap-table#1182]
Is it possible to scroll the footer if the column count exceed the display width?

**Comment by liKe2k1** [comment]
Work arround:

```
$('table').bootstrapTable()
	.on('load-success.bs.table', function (e, data) {
		$('.fixed-table-body').on('scroll', function () {
			$('.fixed-table-footer').scrollLeft($(this).scrollLeft());
		});
	});
```

**Comment by wenzhixin** [comment]
Fixed this issue, #639
